storage/tianmu/index/rc_table_index.cpp

```diff
@@ -21,7 +21,7 @@
 
 #include "common/common_definitions.h"
 #include "core/transaction.h"
-#include "handler/tianmu_handler.h"
+#include "handler/ha_tianmu.h"
```

Comment:
Do you think whether we change all files with `rc_table_xxxx` to `tianmu_table_xxx` is a good or not?

Reply:
Good idea!